Explorer/Assets/DCL/PerformanceAndDiagnostics/Optimization/AdaptivePerformance/Systems/UpdatePhysicsSimulationSystem.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Arch.Core;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using DCL.Profiling;
+using DCL.RealmNavigation;
+using ECS.Abstract;
+using UnityEngine;
+
+namespace DCL.Optimization.AdaptivePerformance.Systems
+{
+    [UpdateInGroup(typeof(PhysicsSystemGroup))]
+    public partial class UpdatePhysicsSimulationSystem : BaseUnityLoopSystem
+    {
+        private readonly AdaptivePhysicsSettings settings;
+        private readonly Profiler profiler;
+        private readonly ILoadingStatus loadingStatus;
+
+        public UpdatePhysicsSimulationSystem(World _, Profiler profiler, AdaptivePhysicsSettings settings, ILoadingStatus loadingStatus) : base(_)
+        {
+            this.settings = settings;
+            this.profiler = profiler;
+            this.loadingStatus = loadingStatus;
+        }
+
+        protected override void Update(float t)
+        {
+            switch (settings.Mode)
```

Comment:
Since mode never changes during runtime, you can simply not inject this system when it's not needed.